sale_order_import/models/res_company.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    picking_policy = fields.Selection(
+        [
+            ("direct", "Deliver each product when available"),
+            ("one", "Deliver all products at once"),
+        ],
+        string="Shipping Policy",
+    )
```

Comment:
@AungKoKoLin1997 
Why does the wizard show another string instead of this one?
![2023-02-02_19h23_49](https://user-images.githubusercontent.com/101688144/216300981-51421478-e236-4977-aa5f-d5d8c9e70d48.png)

Reply:
@kanda999 
In wizard form, I take the options from existing field of sale order https://github.com/odoo/odoo/blob/51ff14a56b909bdf22c9f1c4540347655578e1f4/addons/point_of_sale/models/pos_config.py#L164-L165.
I add the new commit to make the same in `res.company` form. [e8785d7](https://github.com/qrtl/rmm-custom/pull/52/commits/e8785d70907a0020ae4715509608520c76d514b6)

Reply:
@AungKoKoLin1997 
The selection item in Shipping Policy field can not translate.  
Could you please figure it out?

Reply:
Yes. I will try to figure out.

Reply:
@kanda999  I don't see the way to translate the field `picking_policy`. Shall we revert to the original selection field? Getting the options from existing field is stuck at the language translation cause these values are dynamic and translation file can't generate these dynamic string. How do you think?

Reply:
@AungKoKoLin1997  I think it would be better to revert to the original selection field as translation is necessary.
In wizard form, I don't get the point why do you take the options from existing field.  Can we use the original selection field in the wizard?

Reply:
At first, I use original selection field and then I tried to get from existing field because of this [one](https://github.com/qrtl/rmm-custom/pull/52#discussion_r1088563328). But I didn't expect about translation issue.

Reply:
I got it. Can you take care please?  I think that the string should be updated as suggestion.

Reply:
Done!